test/e2e/volume_provisioning.go

```diff
@@ -122,25 +122,9 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			testDynamicProvisioning(c, claim)
 		})
 	})
-
-	framework.KubeDescribe("DynamicProvisioner Alpha", func() {
```

Comment:
I mean to have a test case that checks that the pvc got bound for the default storageclass on the supported cloud providers.